### PR TITLE
#163519231 Social Share components

### PR DESCRIPTION
**Pull request by fabzer0**
#### What does this PR do?
   - This PR displays social share components on every article that is created
   - This should allow users to share the articles with their social sites

#### How should this be manually tested?
1. Run the following commands in Terminal:
```
    git clone https://github.com/andela/ah-jumanji-frontend.git
    git checkout ft-create-social-media-components-163519231
    touch .env
```
2.  Edit the `.env` by adding this:
```
BASE_URL=https://ah-jumanji-staging.herokuapp.com
```
3. Continue in Terminal
```
    npm i && npm audit fix && npm start
```
4. After logging in, click on the `More ...` button on an article to view the icons

#### What are the relevant pivotal tracker stories?
[#163519231](https://www.pivotaltracker.com/story/show/163519231)

#### Screenshots (if appropriate)
screenshot 2019-01-29 at 19 22 32

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

**Review by GransonO — APPROVED** [review]
Looks good. Adds no issues to the current repo state. However, some styling would make it better. Colored icons would be it

**Review by ghost — CHANGES_REQUESTED** [review]
image

I think this would look better if we:
1. restyle the icons so that they match the sites they represent (i.e., use `Facebook's blue` for the Facebook icon, and etc)
2. convert the icons into `anchor tags (links)`. They are after all pointers to other sites.
`OR, instead of 2. above,`
use `css` to change the cursor to a `pointer` when a user hovers over the icons.

**Comment by fabzer0** [comment]
> image
> 
> I think this would look better if we:
> 
> 1. restyle the icons so that they match the sites they represent (i.e., use `Facebook's blue` for the Facebook icon, and etc)
> 2. convert the icons into `anchor tags (links)`. They are after all pointers to other sites.
>    `OR, instead of 2. above,`
>    use `css` to change the cursor to a `pointer` when a user hovers over the icons.

This has been resolved

**Review by EmmanuelBeja — COMMENTED** [review]
could you center the icons and have two <hr> tags enclosing them.

**Review by ghost — APPROVED** [review]
👍

**Comment by fabzer0** [comment]
> could you center the icons and have two
> 
> tags enclosing them.

What tags

**Comment by fabzer0** [comment]
> could you center the icons and have two
> 
> tags enclosing them.

Would centring the articles look nice really? That would mean I have to centre the *Share Article*  title too of which I don't think is a nice idea

**Comment by EmmanuelBeja** [comment]
> > could you center the icons and have two
> > tags enclosing them.
> 
> Would centring the articles look nice really? That would mean I have to centre the _Share Article_ title too of which I don't think is a nice idea

This is what i had in mind.

screen shot 2019-01-30 at 09 23 53

File: src/apps/SocialShare/components/SocialShareIcons.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class SocialIcons extends React.Component {
+
+
+  render() {
+    return (
+      <div>
+        <span id="share-text">Share Article</span>
+        <div className="dropdown-item more btn-link" data-toggle="modal" data-target="#social-icons">
```
Comment (EmmanuelBeja) [inline src/apps/SocialShare/components/SocialShareIcons.js:11]: You should remove the code:
data-toggle="modal" data-target="#social-icons"

it is meant for modals.

**Pull request closed without merging**